server/src/com/cloud/configuration/Config.java

```diff
@@ -1900,9 +1900,9 @@
             Boolean.class,
             "execute.in.sequence.network.element.commands",
             "false",
-            "If set to true, DhcpEntryCommand, SavePasswordCommand, UserDataCommand, VmDataCommand will be synchronized on the agent side."
-                + " If set to false, these commands become asynchronous. Default value is false.",
-            null),
+                    "If set to true, DhcpEntryCommand, SavePasswordCommand, VmDataCommand will be synchronized on the agent side."
```

Comment:
@GabrielBrascher can you fix indentations changes in this class/file?

Reply:
Fixed, thanks @rhtyd.